edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -562,73 +562,73 @@ Above all, the competitor limit must not be exceeded for any reason. Competitor
-  - Finishing the solve
-    - The first step is to evaluate whether the cube is solved
-    - Do not touch the puzzle, use only your eyes
+  - How to judge during the solve.
+    - Pay attention. Don't cube, use your phone, or engage in conversation.
+    - Don't distract the competitor during the solve. Sit still and be quiet.
+    - Watch the competitor for any regulation violations.
+    - Pay close attention to how the competitor stops their timer, and make sure they are using their hands with palms flat and facing down.
+  - Finishing the solve.
+    - The first step is to evaluate whether the cube is solved.
+      - Do not touch the puzzle, use only your eyes.
     - How to evaluate the cube for +2s:
-      - On a 3x3x3, explain that a single 90 degree turn (e.g. U or U’) is a +2
-      - Explain that a double move (e.g. U2 or R2) is also a +2, and not considered 2 separate moves
-      - Explain that a slice move (e.g. M or M2) is considered 2 moves, and is thus a DNF
-      - If the cube is close to the limit for a +2 penalty (45 degrees), and the judge cannot determine the final state, call over a Delegate to make the final decision
-    - Explain that a corner twist is a DNF
-    - Explain how missing pieces affect the solved state
-      - The complexity of regulation 5b5 can make this difficult for competitors to understand. Emphasize that judges can call over a Delegate if they are not certain about how to rule certain missing pieces
-    - How to write the final time
-      - Emphasize legible handwriting
-    - How to write the final time if there are penalties
-    - Ask the competitor to sign for the attempt
-  - What to do after the attempt is over (this may vary depending on how your competition is being run)
+      - Explain that a single turn (e.g. U or U') is a +2 on an NxNxN puzzle
+      - Explain that a "double move" (e.g. U2 or R2) is also a +2, and not considered two separate moves
+      - Explain that a slice move (e.g. M or M2) is considered as two outer moves, and is thus a DNF
+      - Tell them that if the cube is close to the limit for a +2 penalty (e.g. 45 degrees on NxNxN puzzles), and the judge cannot determine the final state, they should call over a Delegate to make the final decision
+    - Explain that a corner twist is a DNF.
+    - Explain how missing pieces affect the solved state.
+      - The complexity of regulation 5b5 can make this difficult for competitors to understand. Emphasize that judges can call over a Delegate if they are not certain about how to rule certain missing pieces.
+    - How to write the final time.
+      - Emphasize legible handwriting.
+    - How to write the final time, if there are penalties.
+    - Ask the competitor to sign for the attempt.
+  - What to do after the attempt is over (this may vary depending on how your competition is being run).
```

Comment:
I would recommend:
"What to do after the attempt is over.  This may vary depending on how your competition is being run."